chart/kubeapps/templates/apprepository-jobs-postupgrade.yaml

```diff
@@ -49,10 +49,17 @@ spec:
             - --database-name=assets
           env:
             - name: DB_PASSWORD
+              {{- if .Values.postgresql.existingSecret }}
```

Comment:
Wouldn't this be a bit DRYer if we just put the conditional around the `valueFrom.secretKeyRef.name`? Also in the other resources below.